gulpfile.js

```diff
@@ -240,25 +242,26 @@ async function generateReactWebviewsBundle() {
 			typecheckPlugin()
 		],
 		sourcemap: prod ? false : 'inline',
-		metafile: !prod,
+		metafile: true,
 		minify: prod,
 		minifyWhitespace: prod,
 		minifyIdentifiers: prod,
+		format: 'esm',
```

Comment:
The default format for esbuild is 'cjs'. However, because esbuild doesn't support code splitting with 'cjs', I had to switch to 'esm'.